### DISCUSSION TITLE
TX EU/UK TOUCH V2.0.0 2021.06.24 (Sonoff T0 1 Gang EU) GPIO0

### DISCUSSION THREAD

**Discussion by Bel-Kin** [arendst/Tasmota#14428] [Q&A] [answered]
Hello everyone! Who can tell on which resistor the GPIO is located in order to flash the firmware? Thanks!
![sonoff_T0_TX_V2](https://user-images.githubusercontent.com/16318605/149475478-0b8b66da-a34f-4d9b-8efa-81397b62db5c.jpg)!

**Comment by sfromis** [comment, marked as answer]
Instead of going crazy hunting for resistors, it is much easier to ground gpio0 directly on the ESP8285 chip. It is number two from the right, on the bottom edge as seen in your image above. As it is no problem if you also ground gpio4 (which is the one to the right of gpio0), you can just rest a pin between them to ground both.
https://tasmota.github.io/docs/Pinouts/

**Reply by mikizvi** [reply]
Thanks.  That worked.  Was a little concerned about not touching a neighboring pin: thanks a span for pointing out that you could rest the probe between the gpio0 and the one on its right.

**Comment by Bel-Kin** [comment]
Working! Thank you!!!

**Reply by henkiejan1** [reply]
I got the same question.  GND connected to TP0 (KEY 1) on the backside of the mainboard. Hold it connected when flashing and that's it!

**Reply by sfromis** [reply]
Yeah, while it works just fine if you locate where they hid gpio0 on each variant (what a mess), it is much simpler to just use the stable pin positions on the chip itself, when it does not take being very exact to only touch one pin. Of course, if you *like* hunting positions on the board, that's great too :-D
